Added the About Page (#35)
pythonindia/inpycon2020: about.md

---
layout: default
title: About Us
---

#  <span style="color:#EB090D">Divided by COVID. Connected by Python</span>

## What is PyCon? 

PyCon India is the largest annual gathering of Python aficionados and experts in India where Pythonistas come from around the world to celebrate their use of Python, and share their knowledge and experience. It is a volunteer-driven conference aimed to foster the adoption, use, and development of the Python programming language. The 12th edition of PyCon was supposed to take place in the beautiful city of Bangalore. But due to the current situation, the conference will take place virtually from 2nd October to 5th October 2020. With this move, this year, we are making the conference bigger, better, and more accessible to Pythonistas all over the world.

## What happens during the conference?

PyCon India 2020 consists of 3 events rolled into one mega event for 3 days of learning and fun:

> * __Conference Days:__ {Dates here} will be the Conference Days. On the conference days, people will be able to connect with other Pythonistas, discuss their experiences, present their ideas, and attend the talks.

Author: I would need specific dates for the different events at the conference.

> * __Workshop Day:__ Workshop Day will be on {Date}, 2020 (Please note that Workshop overlaps with Devsprints). On the workshop day, people can attend hands-on workshops, and learn about a topic they have been excited about, but didn't get the time or resources to learn about, from an expert.

Author: Also here

> * __Developer Sprints:__ The Developer Sprints will be on {Dates}, 2020. During the Devsprints, people can contribute to open source projects, guided by mentors having experience with open-source and their projects. This is a great opportunity for people to start contributing to Python and other open-source projects. You can learn more about Devsprints from [our blog](https://in.pycon.org/blog/2019/understanding-devsprints.html).

Author: And here

## Want to submit a talk/workshop for the conference?

With PyCon India 2020 being online this year, we are introducing preview videos to our proposal submission workflow. Preview videos are you talking about your proposal, topics you intend to cover in the talk, and how you intend to cover them. Participants are strongly suggested to upload links of their preview videos while submitting their proposals. It will be immensely helpful for reviewers to go through preview videos and take that into account before making final decisions on your talk proposal. You can check out our [CFP page](https://in.pycon.org/cfp/2020/proposals/) and [blog](https://in.pycon.org/blog/2020/2020-workshop-cfp-announcement.html) for more details.

## Who are the Organizers and what is our vision?

> ### <span style="color:#F6B700">Organized By the people, For the people</span>

Reviewer: ```suggestion
### <span style="color:#F6B700">Organized by the people, for the people</span>

```

> PyCon India is run every year by Python users and community members all across India to bring their experience and expertise together to share with everyone. The conference is organized entirely by volunteers all the way from planning it, finding the speakers, managing the venue, to completing the event successfully. We have a large team of very enthusiastic and experienced volunteers coming from various Python communities all across India. 

Reviewer: ```suggestion
PyCon India is organized every year by Python communities and user groups all across India as a way to pool their experience and expertise into making a more successful, more accessible conference every year. The conference is organized entirely by volunteers all the way from planning, to finding the speakers, to managing the event successfully. We have a very large distributed team of enthusiastic and experienced volunteers coming from various Python communities all across India. 

```

---
layout: default
title: About Us
---

> #  <span style="color:#EB090D">Divided by COVID. Connected by Python</span>

Reviewer 2: Just a thought.

```suggestion
#  <span style="color:#EB090D">Divided by a pandemic. Connected by Python</span>

```

Author: Sound better tbh

> ## What is PyCon? 

Reviewer 2: ```suggestion
## What is PyCon India? 

```

> 

Reviewer 2: nit: any reason for the double empty lines?

Author: yeah, as you can see in https://in.pycon.org/blog/2020/2020-call-for-volunteers.html, I feel there's not enough space between the headings and the content. So, that's why I've used double lines for the headings.

Author: So, should I keep the double lines, or make them single line?

Reviewer 2: Yeah. That makes sense.

Overall, I feel the paragraph spacing needs to be fixed. Although this is an unrelated change.

Reviewer 2: @Sc0rpi0n101 Please keep them for now. We will review later.

PyCon India is the largest annual gathering of Python aficionados and experts in India where Pythonistas come from around the world to celebrate their use of Python, and share their knowledge and experience. It is a volunteer-driven conference aimed to foster the adoption, use, and development of the Python programming language. The 12th edition of PyCon was supposed to take place in the beautiful city of Bangalore. But due to the current situation, the conference will take place virtually from 2nd October to 5th October 2020. With this move, this year, we are making the conference bigger, better, and more accessible to Pythonistas all over the world.

## What happens during the conference?

> 

Reviewer 2: nit: same as above.

* __Developer Sprints:__ The Developer Sprints will be on Oct 4-5, 2020. During the Devsprints, people can contribute to open source projects, guided by mentors having experience with open-source and their projects. This is a great opportunity for people to start contributing to Python and other open-source projects. You can learn more about Devsprints from [our blog](https://in.pycon.org/blog/2019/understanding-devsprints.html).

## Want to submit a talk/workshop for the conference?

With PyCon India 2020 being online this year, we are introducing preview videos to our proposal submission workflow. Preview videos are you talking about your proposal, topics you intend to cover in the talk, and how you intend to cover them. Participants are strongly suggested to upload links of their preview videos while submitting their proposals. It will be immensely helpful for reviewers to go through preview videos and take that into account before making final decisions on your talk proposal. You can check out our [CFP page](https://in.pycon.org/cfp/2020/proposals/) and [blog](https://in.pycon.org/blog/2020/2020-workshop-cfp-announcement.html) for more details.

> ## Who are the Organizers and what is our vision?

Reviewer 2: ```suggestion
## Who are the organizers and what is our vision?

```

### <span style="color:#F6B700">Organized by the people, for the people</span>

> 

Reviewer 2: nit: double empty lines.

PyCon India is organized every year by Python communities and user groups all across India as a way to pool their experience and expertise into making a more successful, more accessible conference every year. The conference is organized entirely by volunteers all the way from planning, to finding the speakers, to managing the event successfully. We have a very large distributed team of enthusiastic and experienced volunteers coming from various Python communities all across India. 

> 

Reviewer 2: nit: double empty lines.

Our Vision is to bring the cumulative knowledge and experience of our community members to the public. We want to create a responsible and inclusive ecosystem where Pythonistas from diverse backgrounds can come together to share their stories and gain something from other's experiences. 

> The python programming language has become an essential tool for people from various backgrounds and careers, including programmers, mathematicians, doctors, scientists, security researchers, hardware designers, and many more. We aim to ensure the conference can reflect the diversity of our community and facilitate the exchange of ideas, knowledge, and experiences enriching the community. 

Reviewer 2: ```suggestion
The Python programming language has become an essential tool for people from various backgrounds and careers, including programmers, mathematicians, doctors, scientists, security researchers, hardware designers, and many more. We aim to ensure the conference can reflect the diversity of our community and facilitate the exchange of ideas, knowledge, and experiences enriching the community. 

```
